cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -58,38 +58,28 @@
 # - ``RTIConnextDDS::cpp2_api``
 #   The nddscpp2 library if found (nddscore and nddsc will be linked as part
 #   of this target).
+# - ``RTIConnextDDS::metp``
+#   The METP library if found (nddsmetp).
+# - ``RTIConnextDDS::rtixml2``
+#   The RTI XML2 library if found (rtixml2).
+# - ``RTIConnextDDS::apputils_c``.
+#   The APP Utils C library (rtiapputils).
+# - ``RTIConnextDDS::rtisqlite``.
+#   The RTI SQLite library (rtisqlite).
 # - ``RTIConnextDDS::distributed_logger_c``
 #   The C API library for Distributed Logger if found.
 # - ``RTIConnextDDS::distributed_logger_cpp``
 #   The CPP API library for Distributed Logger if found.
-# - ``RTIConnextDDS::metp``
-#   The METP library if found (nddsmetp).
-# - ``RTIConnextDDS::routing_service_infrastructure``
-#   The infrastructure library for Routing Service if found.
-# - ``RTIConnextDDS::routing_service_c``
-#   The C API library for Routing Service if found (includes
-#   rtiroutingservice, rtirsinfrastructure, rtixml2 and rticonnextmsgc. Also
-#   nddsmetp and rticonnextmsgc if found).
-# - ``RTIConnextDDS::routing_service``
-#   The same as RTIConnextDDS::routing_service_c. Maintained for backward
-#   compatibility.
-# - ``RTIConnextDDS::routing_service_cpp``
-#   The same as RTIConnextDDS::routing_service_c but adding the CPP libraries.
-# - ``RTIConnextDDS::routing_service_cpp2``
-#   The same as RTIConnextDDS::routing_service_c but adding the CPP2 libraries.
-# - ``RTIConnextDDS::assign_transformation``
-#   The assign transformation library if found (includes rtirsassigntransf and
-#   rtiroutingservice).
-# - ``RTIConnextDDS::monitoring``
-#   The Monitoring library if found.
-# - ``RTIConnextDDS::security_plugins``
-#   The security plugins libraries if found (nddssecurity).
 # - ``RTIConnextDDS::messaging_c_api``
 #   The Request Reply C API library if found (rticonnextmsgc).
 # - ``RTIConnextDDS::messaging_cpp_api``
 #   The Request Reply CPP API library if found (rticonnextmsgcpp).
 # - ``RTIConnextDDS::messaging_cpp2_api``
 #   The Request Reply CPP2 API library if found (rticonnextmsgcpp2).
+# - ``RTIConnextDDS::security_plugins``
+#   The security plugins libraries if found (nddssecurity).
```

Comment:
`The Security Plugins libraries if found (nddssecurity).`

(since it's also a product name, and other modules, like "Monitoring library" also use initial caps)